apps/spline.c

```diff
@@ -62,13 +103,32 @@ static void _apps_spline_paint(apps_spline_t *spline)
     twin_path_destroy(path);
 }
 
+static void _apps_spline_button_signal(maybe_unused twin_button_t *button,
+                                       twin_button_signal_t signal,
+                                       void *closure)
+{
+    if (signal != TwinButtonSignalDown)
+        return;
+
+    apps_spline_t *spline = closure;
+    spline->n_points = (spline->n_points == 3) ? 4 : 3;
+    _init_control_point(spline);
+    _twin_widget_queue_paint(&spline->widget);
+}
+
+#define D(x) twin_double_to_fixed(x)
```

Comment:
The `D(x) twin_double_to_fixed(x)` has already been defined. Another question is: why not use `twin_int_to_fixed()` in `twin_button_create()`

Reply:
I think both are OK. I use `D(x)` just for simplicity.

Reply:
There is `#define D(x) twin_double_to_fixed(x)` in line 14 of apps/spline.c